extensions/sql-database-projects/src/test/dialogs/addDatabaseReferenceDialog.test.ts

```diff
@@ -22,28 +22,38 @@ describe('Add Database Reference Dialog', () => {
 	});
 
 	beforeEach(function (): void {
-		const dataWorkspaceMock = TypeMoq.Mock.ofType<dataworkspace.IExtension>();
-		dataWorkspaceMock.setup(x => x.getProjectsInWorkspace(TypeMoq.It.isAny(), TypeMoq.It.isAny())).returns(() => Promise.resolve([]));
-		sinon.stub(vscode.extensions, 'getExtension').returns(<any>{ exports: dataWorkspaceMock.object });
+		// const dataWorkspaceMock = TypeMoq.Mock.ofType<dataworkspace.IExtension>();
+		// dataWorkspaceMock.setup(x => x.getProjectsInWorkspace(TypeMoq.It.isAny(), TypeMoq.It.isAny())).returns(() => Promise.resolve([]));
+		// sinon.stub(vscode.extensions, 'getExtension').withArgs('Microsoft.data-workspace').returns(<any>{ exports: dataWorkspaceMock.object });
```

Comment:
Circle back to this with Charle's solution for mocking out the other extensions.